packages/flutter_tools/test/general.shard/build_system/targets/linux_test.dart

```diff
@@ -133,6 +135,9 @@ void main() {
     expect(output.childFile('kernel_blob.bin'), exists);
     expect(output.childFile('AssetManifest.json'), exists);
     expect(output.childFile('version.json'), exists);
+    final String versionFile = output.childFile('version.json').readAsStringSync();
+    expect(versionFile, contains('"version":"2.0.0"'));
```

Comment:
why is the key here `version`?

Reply:
`version` key is used for representing `Project` version.
getVersionInfo method from Project is responsible for creating content for version.json and this method is using that key.
Maybe we can move these keys to some const values.

Reply:
does `kBuildName` somewhere get mapped to `version`? I'm looking at the diff and it looks like you're asserting that adding `"BuildName": "2.0.0"` to the defines result in `"version": "2.0.0"` being written to `version.json`, which is a little surprising to me.

Reply:
Note, if this is the case, it's not a problem with your change perse, but surprising.

Reply:
No, no. `kBuildName` is used in `BuildInfo` defines, for transfering data from build info to actual build.
You can see it here in action:
`    if (defines.containsKey(kBuildName)) {
      versionInfo['version'] = defines[kBuildName];
}`

Reply:
So why not call this `build_name` in the version.json?

Reply:
That would require changes to packages that already use it as ```version. ``` I don't think we should change it.